.github/workflows/preview.yaml

```diff
@@ -50,12 +50,30 @@ jobs:
         shell: bash
       - name: Install Vercel CLI
         run: npm install --global vercel@latest
+      - name: Generate preview domain names
+        id: domains
+        run: |
+          REF="${{ github.head_ref }}"
+          CLEAN_REF="${REF//[^a-z0-9\-]/x}"
```

Comment:
Shove this into the step output for the `database` step so we have one source of truth please